src/components/input-panel/compiled-spec-header/index.tsx

```diff
@@ -73,6 +127,34 @@ class CompiledSpecDisplayHeader extends React.PureComponent<Props> {
             </>
           )}
 
+          {this.props.sidePaneItem === SIDEPANE.Config && (
+            <>
+              <ArrowUpCircle
+                data-tip
+                data-for="mergeConfig"
+                onClick={e => {
+                  e.stopPropagation();
+                  this.handleMergeConfig(e);
+                }}
+              />
+
+              <ArrowDownCircle
+                data-tip
+                data-for="extractConfig"
+                onClick={e => {
+                  e.stopPropagation();
+                  this.handleExtractConfig();
+                }}
+              />
+              <ReactTooltip id="mergeConfig" effect="solid">
+                <span style={{ textTransform: 'none' }}>Merge config into spec</span>
```

Comment:
```suggestion
                <span>Merge config into spec</span>
```

Reply:
Do we want all caps?

Reply:
Oh, no, we don't. I didn't know that's the default.